pkg/x/muxer/muxer.go

```diff
@@ -96,10 +97,15 @@ type (
 )
 
 func (m *muxer) getFunction(token string) server {
+	// TODO: Verify this is safe to do
```

Comment:
It is not safe to do. We need to recommend what we want long-term here, such as making sure dispatchTable is populated appropriately (precomputed), or falling back on dynamically dispatching if an entry in dispatchTable is not sufficient. This would involve querying sub-providers to find one that supports this function.